src/routes/console/project-[project]/messaging/providers/create.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+    import { onDestroy } from 'svelte';
+    import { Wizard } from '$lib/layout';
+    import type { WizardStepsType } from '$lib/layout/wizard.svelte';
+    import Step1 from './wizard/step1.svelte';
+    import Step2 from './wizard/step2.svelte';
+    import Step3 from './wizard/step3.svelte';
+    import Step4 from './wizard/step4.svelte';
+    import { sdk } from '$lib/stores/sdk';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { addNotification } from '$lib/stores/notifications';
+    import { goto } from '$app/navigation';
+    import { base } from '$app/paths';
+    import { project } from '../../store';
+    import { wizard } from '$lib/stores/wizard';
+    import { provider, providerParams } from './wizard/store';
+    import { ID } from '@appwrite.io/console';
+
+    async function create() {
+        try {
+            let response = { $id: '', name: '' };
+            const providerId = $providerParams[$provider].providerId || ID.unique();
+            switch ($provider) {
```

Comment:
@TorstenDittmann, this is going to be long and annoying because there's a different method for each provider. Any thoughts on how to improve this?

Reply:
Torsten: for now, let's go with this.